apps/target-service/src/main.ts

```diff
@@ -1,23 +1,25 @@
 import { NestFactory } from '@nestjs/core';
-import { TargetServiceModule } from './target-service.module';
 import { NestExpressApplication } from '@nestjs/platform-express';
 import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 
+import { TargetModule } from './target.module';
+import { attachKafka } from '@app/kafka';
+import { ValidationPipe } from '@nestjs/common';
+
 async function bootstrap() {
-  const app =
-    await NestFactory.create<NestExpressApplication>(TargetServiceModule);
+  const app = await NestFactory.create<NestExpressApplication>(TargetModule);
 
   // Swagger / OA docs
   const config = new DocumentBuilder()
-    .setTitle('Clock Service')
-    .setDescription('The clock service API description')
+    .setTitle('Target Service')
```

Comment:
```suggestion
    .setTitle('Target Service')
    .setDescription('The target service API description')
```
Voor consistentie :)

Reply:
Die description slaat nergens op. Voor de consistentie moeten ze allemaal weg :)